sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md

```diff
@@ -319,12 +319,9 @@ This example enables users to share with all external collaborators except for t
 ```powershell
 Set-SPOTenant -EnableVersionExpirationSetting $true
 ```
+The `EnableVersionExpirationSetting` parameter is no longer active, this feature is now automatically enabled for each tenant. Setting `EnableVersionExpirationSetting` to false would not disable the feature.
+Learn more about the Version History Setting at https://learn.microsoft.com/en-us/sharepoint/document-library-version-history-limits
```

Comment:
Add a name for the link? Like " [Read the terms of service](https://aka.ms/versioning-termsofservice)"

Reply:
+1

then please use a relative link, so the right locale can be used at runtime. the link to use is the value after the locale, in this case `/sharepoint/document-library-version-history-limits`